drivers/gpu/drm/panel/Kconfig

```diff
@@ -480,6 +480,16 @@ config DRM_PANEL_SAMSUNG_S6D27A1
 	  This panel can be found in Samsung Galaxy Ace 2
 	  GT-I8160 mobile phone.
 
+config DRM_PANEL_SAMSUNG_S6E3FA0_FULL
```

Comment:
you don't need _FULL here, that was my hack to avoid future conflicts with separate drivers (I have one full unified)

Reply:
Removed _FULL